### PR TITLE
Make python package

### PR DESCRIPTION
**Pull request by andrebask**
Add setup.py to package the project with setuptools

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    install_requires=[
+        'Sphinx==1.8.5 ; python_version < "3.5"',
+        'Sphinx==2.0.1 ; python_version >= "3.5"',
+        'sphinx-rtd-theme==0.4.3',
+        'requests==2.21.0',
+        'requests-mock==1.6.0',
+        'pytest==4.4.1',
+        'alpha-vantage==2.1.0',
+        'docutils==0.14',
+        'm2r==0.2.1',
```
Comment (ilcardella) [inline setup.py]: Would this replace the need for the `requirements.txt` files? and probably the installation script too?

**Comment by andrebask** [comment]
I think they may still be useful for development. Actually in the setup.py
there should only be the dependencies for running the application as an end
user, but I'm not sure which is which :)

File: setup.py (hunk 1 above)
Comment (andrebask) [reply setup.py]: I think they may still be useful for development. Actually in the setup.py
there should only be the dependencies for running the application as an end
user, but I'm not sure which is which :)

Comment (ilcardella) [reply setup.py]: Then only alpha-vantage is required here.

We should also update the renovate.json to work with setup.py

**Pull request merged**